### PR TITLE
Include gemm_helper.h

### PR DESCRIPTION
**Pull request by oliviajain**
**Description**: Small fix to allow building ACL EP

**Motivation and Context**
- Resolve missing reference to gemm_helper.h

### CODE REVIEW

File: onnxruntime/core/providers/acl/math/gemm.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #include "core/util/math.h"
 #include "core/util/math_cpuonly.h"
 #include "core/providers/cpu/math/gemm.h"
+#include "core/providers/cpu/math/gemm_helper.h"
```
Comment (yuslepukhin) [inline onnxruntime/core/providers/acl/math/gemm.h:11]: So, this includes a header into another header which I feel may disseminate it too wide. Where exactly gemm_helper.h definitions are needed? My understanding they are not needed in gemm.h but some place else. We want to include headers into .cc files where they are required.

**Review by jywu-msft — APPROVED** [review]
Thanks!

File: onnxruntime/core/providers/acl/math/gemm.h (hunk 1 above)
Comment (jywu-msft) [reply onnxruntime/core/providers/acl/math/gemm.h:11]: it's needed for the Compute() method.
probably better to move the Compute() to .cc and include the headers from there.

Comment (oliviajain) [reply onnxruntime/core/providers/acl/math/gemm.h:11]: Should I make that change in this PR? I noticed you approved the current fix.

Comment (jywu-msft) [reply onnxruntime/core/providers/acl/math/gemm.h:11]: we can refactor later. let's just fix the ACL build for now.

**Pull request merged**